src/js/main.js

```diff
@@ -0,0 +1,59 @@
+const form = document.getElementById('registrationForm');
+const fullName = document.getElementById('fullName');
+const phoneNumber =  document.getElementById('phoneNumber');
+const hideInfo = form.querySelector('#hidden-info');
+const privateInfo = document.getElementById('privateInfo');
+const inputRegion = form.querySelector('#region');
+const city = form.querySelector('select#city');
+const cities = form.querySelector('#cities');
+const submit = form.querySelector('input#submit');
+
+fullName.addEventListener('blur', () => {
+  console.log(fullName);
+  const regStr = /^[А-ЯA-Z][а-яa-zА-ЯA-Z\-]{0,}\s[А-ЯA-Z][а-яa-zА-ЯA-Z\-]{1,}(\s[А-ЯA-Z][а-яa-zА-ЯA-Z\-]{1,})?$/;
+    if (regStr.test(fullName.value)) {
+      fullName.style.backgroundColor = 'green';
```

Comment:
use classes and add/remove classes instead of manipulating css directly in the js code.